Neos.Flow/Classes/Mvc/Controller/Argument.php

```diff
@@ -180,6 +181,11 @@ public function getDefaultValue()
     public function setValidator(ValidatorInterface $validator): Argument
     {
         $this->validator = $validator;
+        if ($validator !== null && $this->value !== null) {
```

Comment:
Same as with https://github.com/neos/flow-development-collection/pull/1906/files#diff-03d5ab75f1321a95bb4043fbb9ca202d25246c9db5c354664f45c23d4a24bde6L217 - we can just have those lines independent of a feature flag setting in the controller, because it will just work as before depending on the order of the `initializeActionMethodValidators()` method call relative to `mapRequestArgumentsToControllerArguments()`. As a bonus (FEATURE), it makes sure that if a controller manually overwrites the validator of an argument, it will be executed.

Reply:
Maybe this could be condensed into a short inline comment!?